### ISSUE TITLE
Android | Remove controller's HUD

### ISSUE THREAD

**Issue by Selfiws** [Hydr8gon/NooDS#99]
I am running the software on the Android with my ps4 controller and it is inconvenient the HUD appearing on the screen since i won't use it, can you create an option to remove it?

**Comment by Selfiws** [comment]
Oh, my back button isn't working, must be a bug. I'm using gestures hand one operation from a Samung Tab, and doesn't do nothing when i swipe to go back. I will try with the regular back to see if it is working.

**Comment by Selfiws** [comment]
Yeah as I suspected, swipe operations doesn't work on the software, just the virtual button's one. That is why i couldn't check that option.

**Comment by Hydr8gon** [comment]
Ah, weird. I don't have gestures on my phone, so I wasn't aware it didn't work. I'll try to look into it at some point.

**Comment by Selfiws** [comment]
Should I create a new issue with the properly bug?

**Comment by Hydr8gon** [comment]
I'll just rename this one, since it contains relevant discussion.
